packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PinModal.tsx

```diff
@@ -37,16 +31,10 @@ export const PinModal = ({ device, ...rest }: PinModalProps) => {
     // 3 cases when we want to show left column
     // 1) and 2) - Setting a new pin: 1 entry, 2nd (confirmation) entry
     // 3) Invalid pin (It doesn't seem to work anymore) instead separate PinMismatchModal is shown
```

Comment:
I am not sure if I understand what this comment is here for, but I assume its relevant to the code you have extracted.

Maybe this comment shall be removed (that would be idea, if code is self-documenting), rephrased or moved to the `usePin` hook you have created

Reply:
Ok, I've migrated it to `usePinModal.ts`